lib/puppet/application/lookup.rb

```diff
@@ -127,11 +102,6 @@ def setup
     Puppet.settings.use :main, :master, :ssl, :metrics
 
     setup_terminuses
-
-    # TODO: Do we need this in lookup? It sets up a write only cache
```

Comment:
It's consistent with the documentation of the method where it says _Sets up a special node cache "write only yaml"_.